### DISCUSSION TITLE
Why are my viewers not follow a raid?

### DISCUSSION THREAD

**Discussion by dedalo88888** [KevinBytesTheDust/CVAmp#164] [Q&A] [answered]
Hello,
First of all, thank you for the great work with this program so simple and so helpful.
I have been using this program for months.
When I have several green instances open on Twitch and I want to RAID another channel, the clients created with this program do not detect me, it always tells me RAID of 3-7 people when I have 20 instances open, I understand that the 3-7 people that Twitch detects are the real ones, but I would like to know if there is any way to be able to make a RAID of 20 instances and for Twitch to detect it as such.

Thanks in advance

**Comment by jlplenio** [comment, marked as answer]
I have not confirmed it myself, but @Grillekkj stated in #138 that 
> CVTB cannot participate in raids, as it does not yet have the twitch login function and raids need the user to be logged in

The first step, login/authentication via token, is on the way. Preview here: https://github.com/jlplenio/crude-twitch-viewer-bot/discussions/163

**Reply by Grillekkj** [reply]
yep, twitch require logged in user to raid
